rero_ils/modules/loans/api.py

```diff
@@ -71,6 +71,15 @@ class Meta:
 
         default_filter = None
 
+    def get_pending_request_count(self, item_pid):
```

Comment:
Interesting... I would put naturally put this in the Loan class, we have plenty of other example in the code. But why not in the Search. To be discuss with the team.

Reply:
This function already exists: 

https://github.com/rero/rero-ils/blob/staging/rero_ils/modules/items/api/circulation.py#L1209-L1218